docs/integrations/aws.md

```diff
@@ -419,9 +419,22 @@ If for any reason you have to use S3A, here are the instructions:
 To ensure integrity of uploaded objects, checksum validations for S3 writes can be turned on by setting catalog property `s3.checksum-enabled` to `true`. 
 This is turned off by default.
 
+### S3 Delete 
+
+When `s3.delete-enabled` is disabled, the objects are not deleted from S3.
+For example, to disable deletion with Spark 3.0, you can start the Spark SQL shell with:
```

Comment:
[minor] can add what is the default value (though it might be obvious) , referring the checksum verification example above

Reply:
I have ddded the default value.

Reply:
I have added the default value.